site/app/views/submission/HomeworkView.php

```diff
@@ -560,7 +560,7 @@ private function renderSubmitBox(LateDays $late_days, Gradeable $gradeable, $gra
         $recent_version_url = $graded_gradeable ? $this->core->buildCourseUrl(['gradeable', $gradeable->getId()]) . '/' . $graded_gradeable->getAutoGradedGradeable()->getHighestVersion() : null;
         $numberUtils = new NumberUtils();
         return $output . $this->core->getOutput()->renderTwigTemplate('submission/homework/SubmitBox.twig', [
-            'course_path' => $this->core->getConfig()->getCoursePath(),
+            'course' => $this->core->getConfig()->getTerm() . '-' . $this->core->getConfig()->getCourse(),
```

Comment:
that's...definitely not confusing at all...    Why not just pass the course and term separately? So the variable "course" actually contains a course name?